getting_started/first_3d_game/03.player_movement_code.rst

```diff
@@ -0,0 +1,280 @@
+.. _doc_first_3d_game_player_movement:
+
+Moving the player with code
+===========================
+
+It's time to code! We're going to use the input actions we created in the last
+part to move the character.
+
+Right-click the *Player* node and select *Attach Script* to add a new script to
+it. In the popup, set the *Template* to *Empty* before pressing the *Create*
+button.
+
+|image0|
+
+Let's start with the class's properties. We're going to define a movement speed,
+a fall acceleration representing gravity, and a velocity we'll use to move the
+character.
+
+::
+
+   extends KinematicBody
+
+   # How fast the player moves in meters per second.
+   export var speed = 14
+   # The downward acceleration when in the air, in meters per second squared.
+   export var fall_acceleration = 75
+
+   var velocity = Vector3.ZERO
+
+These are common properties for a moving body. The ``velocity`` is a 3D vector
+combining a speed with a direction. Here, we define it as a property because
+we're going to build upon its value frame after frame.
```

Comment:
```suggestion
combining a speed with a direction. Here, we define it as a property because
we want to update and reuse its value across frames.
```